pkg/orchestration/wiring/aws/resources.go

```diff
@@ -34,11 +34,16 @@ const (
 
 // All osb-aws-provider resources are 'almost' the same, differing only in the service/plan names,
 // what they need passed in the ServiceEnvironment.
-var ResourceTypes = map[voyager.ResourceType]wiringplugin.WiringPlugin{
-	DynamoDB: wiringutil.StatusAdapter(Resource(DynamoDB, DynamoDBName, DynamoDBClass, DynamoDBPlan, dynamoDbServiceEnvironment, dynamoDbShapes).WireUp),
-	S3:       wiringutil.StatusAdapter(Resource(S3, S3Name, S3Class, S3Plan, s3ServiceEnvironment, s3Shapes).WireUp),
-	Cfn:      wiringutil.StatusAdapter(Resource(Cfn, CfnName, CfnClass, CfnPlan, CfnServiceEnvironment, cfnShapes).WireUp),
-}
+var (
+	emptyVPC = func(location voyager.Location) *oap.VPCEnvironment {
```

Comment:
How do you construct non-empty VPC and make it environment-specific without LegacyConfig?

Reply:
We're passing a function type based on public types. I won't be using know authowire plugins in the close repo.

Reply:
I'm sorry, I don't understand this English :(

Reply:
It's not. Follow the code in the orchestration main in _this_ PR.

Reply:
I'm confused. If you want to construct plugins dynamically, these static plugin variables simply should not exist, at least in the non-test package.

Reply:
I agree with Nail's point. Would be better to move the not-really-ready-to-use instances of plugins (and other "stubby" stuff) into some testing package

Reply:
Think the comment is now outdated.